### PR TITLE
BRITECHSRM-234 Restructure files by feature

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

**Comment by ianlovell** [comment]
I wonder if renaming the files within each (PascalCase for components) subfolders to index.js would be a good idea? To clean up the imports. So "../components/AppHeader/AppHeader.jsx" becomes "../components/AppHeader".

**Comment by lclarksl** [comment]
I thought you needed to include .the file extension when importing jsx files?

**Comment by ianlovell** [comment]
Yes that's true, it means the components would need a `.js` extension. What do you think? It's what Facebook (controversially) do as well - https://github.com/kriasoft/react-starter-kit/issues/52. It appears the only reason is to shorten the references on imports.

Perhaps leave it for now and we can reassess! :shipit:

File: app/components/Dashboard/DashboardSidebar/SessionFilterCheckbox.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Component, PropTypes } from 'react';
 import Checkbox from 'material-ui/lib/checkbox';
-import SessionStatusService from '../services/SessionStatusService';
+import SessionStatusService from '../../../services/SessionStatusService';
```
Comment (csscottlogic) [inline app/components/Dashboard/DashboardSidebar/SessionFilterCheckbox.jsx]: Going up three levels looks a bit fragile to me?

**Comment by csscottlogic** [comment]
As long as the structure means that ```../../../``` makes sense, then :shipit:

**Pull request merged**